field_rrule/demo/res_partner.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
```

Comment:
Since there isn't a cleaner way to do it, I suggest having the demo as a separate module `field_rrule_demo`.
This would allow us to see the module in action on Runbot.